### ISSUE TITLE
Problem with not having to easily Seed Mock Data

### ISSUE THREAD

**Issue by HarshPatel5940** [srm-kzilla/fury#427]
## Suggestion In
- [x] Server and Client Side

### TLDR
Helps new contributors to get started using fury

# Context
Seeding of sample data in MongoDB, why tho? _too much work for me_
- a) First it obvious useCase to easily generate some mock data to easily test something.
- b) It will also help new contributors, here's how
     - What collections is required to get started
     - What type of basic data is needed after the collections are done.

_This will ultimately help everyone understand how fury works, with bit more **clarity**_ and get started quickly in contributing without hassling with initialising the mongoDB.

# How to approach this?
I am leaving this up to you and come up with optimal solutions and idea's to quickly seed all the 3 clients and the backend with a good amount of mock data. You can either use a single script for all these 4 things or separate scripts or surprise us by using mono-repo tools like turbo-Repo.

I would suggest you to create a separate [ISSUE](https://github.com/srm-kzilla/fury/issues/new) to come with a optimal idea for this problem and **linking back the issue with this issue which will help to easily track different ideas.**

**Comment by HarshPatel5940** [comment]
To suggest idea's please create new issues and link them here.
